packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -157,6 +157,8 @@ function SendConfirmation(props: Props) {
   }
 
   const sendOrInvite = (inviteMethod?: InviteBy) => {
+    dispatch(storeLatestInRecents(recipient))
```

Comment:
We should avoid consecutive dispatches whenever possible.
It's fine and even recommended that one single action be handled by multiple reducers.

Sofar the codebase doesn't show a great example of this because we have many actions which are just "setters" and which hence encourages consecutive dispatches.

See https://redux.js.org/style-guide/style-guide#allow-many-reducers-to-respond-to-the-same-action and 
https://redux.js.org/style-guide/style-guide#model-actions-as-events-not-setters

Reply:
ok! I've removed the `storeLatestInRecents` action and just chained it to the `sendAndInvite` action